### ISSUE TITLE
SARIF report import causes 500

### ISSUE THREAD

**Issue by whitespots** [DefectDojo/django-DefectDojo#4940]
**Bug description**
Hi team, we have an issue with SARIF format in `mobsfscan` tool.
You will find the report in attachments below.

**Steps to reproduce**
1. `pip3.9 install mobsfscan`
2. `git clone https://github.com/payatu/diva-android.git diva && cd diva`
3. `mobsfscan --sarif . -o mobsfscan.json`
4. Import it with SARIF scan type and see 500

**Expected behavior**
Imported vulnerabilities

**Deployment method** *(select with an `X`)*
- [X] Docker
- [X] Kubernetes
- [ ] GoDojo
- [ ] setup.bash / legacy-setup.bash

**Sample scan files** (optional)
[sast_mobile_mobsfscan.zip](https://github.com/DefectDojo/django-DefectDojo/files/6983132/sast_mobile_mobsfscan.zip)

**Comment by whitespots** [comment]
An error in CI/CD
![image](https://user-images.githubusercontent.com/65983771/129375422-ef5ce0c2-1d23-465e-a428-4bfe2f431148.png)

**Comment by whitespots** [comment]
UI error
![image](https://user-images.githubusercontent.com/65983771/129376026-7d856369-4b15-4f93-b0d2-93ff696e3554.png)

**Comment by akitibala** [comment]
i guess defectdojo currently does not support sarif format .

**Comment by whitespots** [comment]
It was ok with semgrep some time ago actually, but now even semgrep issues are not parsing correctly. But this is another story

**Comment by valentijnscholten** [comment]
Thanks for reporting. Next time it would be helpful to include the traceback from the logs:
```
Traceback (most recent call last):
  File "/home/valentijn/dd/dojo/engagement/views.py", line 602, in import_scan_results
    test, finding_count, closed_finding_count = importer.import_scan(scan, scan_type, engagement, user, environment, active=active, verified=verified, tags=tags,
  File "/home/valentijn/dd/dojo/importers/importer/importer.py", line 296, in import_scan
    parsed_findings = importer_utils.parse_findings(scan, test, active, verified, scan_type)
  File "/home/valentijn/dd/dojo/importers/utils.py", line 16, in parse_findings
    parsed_findings = parser.get_findings(scan, test)
  File "/home/valentijn/dd/dojo/tools/sarif/parser.py", line 30, in get_findings
    return self.get_items(tree, test)
  File "/home/valentijn/dd/dojo/tools/sarif/parser.py", line 42, in get_items
    item = get_item(result, rules, artifacts, run_date)
  File "/home/valentijn/dd/dojo/tools/sarif/parser.py", line 177, in get_item
    cwes_extracted = get_rule_cwes(result)
  File "/home/valentijn/dd/dojo/tools/sarif/parser.py", line 78, in get_rule_cwes
    cwes.append(int(re.split('-|:', value)[1]))
```

**Comment by valentijnscholten** [comment]
fixed in dev / 2.2.0
